src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -310,22 +300,22 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
       configuration: scenarioConfig,
     };
   }
-
-  private updatePriorityWeightsFormControls(): void {
-    const priorities: string[] = this.nameFormGroup.get('priorities')?.value;
-    const priorityWeightsForm: FormGroup = this.formGroups[3].get(
-      'priorityWeightsForm'
-    ) as FormGroup;
-    priorityWeightsForm.controls = {};
-    priorities.forEach((priority) => {
-      const priorityControl = this.fb.control(1, [
-        Validators.required,
-        Validators.min(1),
-        Validators.max(5),
-      ]);
-      priorityWeightsForm.addControl(priority, priorityControl);
-    });
-  }
+  //
+  // private updatePriorityWeightsFormControls(): void {
```

Comment:
Ah, this one I believe we can remove. I kept it around initially in case we ended up using a formgroup to organize the priority weights but that didn't end up happening